Mathlib/Data/Set/Finite/Parity.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2024 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+import Mathlib.Data.Set.Card
+import Mathlib.Data.Set.Finite
+
+/-!
+# Finite sets and parity of their cardinality
+
+This file contains lemmas concerning finite sets and parity of their cardinality.
+-/
+
+namespace Set.Finite
+
+variable {V : Type _} {s : Set V}
+
+lemma one_lt_ncard_of_nonempty_of_even (hs : Set.Finite s) (hn : Set.Nonempty s)
+    (he : Even (Nat.card s)) : 1 < s.ncard := by
+  have : s.ncard ≠ 0 := by
+    intro h
+    rw [@Set.nonempty_iff_ne_empty] at hn
+    exact hn <| (Set.ncard_eq_zero hs).mp h
+  have : s.ncard ≠ 1 := by
+    intro h
+    rw [Set.Nat.card_coe_set_eq, h] at he
+    simp at he
+  omega
+
+lemma two_of_even_of_nonempty (hs : Set.Finite s) (hn : Set.Nonempty s) (he : Even (Nat.card s)) :
```

Comment:
This lemma is very specific and doesn't seem much easier to apply that the one with `two_le`.

Reply:
Fair enough, removed it:)